shaders/src/common_getters.glsl

```diff
@@ -24,7 +24,12 @@ highp mat4 getViewFromClipMatrix() {
 
 /** @public-api */
 highp mat4 getClipFromWorldMatrix() {
+#if defined(VARIANT_HAS_INSTANCED_STEREO)
+    int eye = instance_index % 2;
+    return frameUniforms.clipFromEyeMatrix[eye] * frameUniforms.eyeFromWorldMatrix[eye];
```

Comment:
I'm assuming we do need both matrices in the shader and that's why we can pre-concatenate them (unless we create even more matrices).

Reply:
I think we eventually want both `clipFromEyeMatrix` and `eyeFromWorldMatrix`. This way, a user can get "eye" space through the `eyeFromWorldMatrix`.